rest/access_test.go

```diff
@@ -182,20 +193,20 @@ func TestStarAccess(t *testing.T) {
 	//
 
 	// Create a user:
-	fran, err := a.NewUser("fran", "letmein", channels.BaseSetOf(t, "*"))
+	fran, _ := a.NewUser("fran", "letmein", channels.BaseSetOf(t, "*"))
```

Comment:
It's great that you caught this was unused! but let's add `require.NoError(t, err)` since everything after this test will fail if new user doesn't work